AirCastingTests/Mocks/APIClientMock.swift

```diff
@@ -7,9 +7,12 @@ import Combine
 @testable import AirCasting
 
 final class APIClientMock: APIClient {
-    var requestTaskStub: ((_ request: URLRequest, _ completion: (Result<(data: Data, response: HTTPURLResponse), Error>, URLRequest) -> Void) -> Void)!
+    var callHistory: [URLRequest] = []
```

Comment:
There is a reason for the stub to exist. also to test if the request was even called. I think its a valid case and should be used inside test case

Reply:
The stub was not removed, it's still there :) github made this line red because I moved it one line down (and added the `@escaping` keyword) 🤷‍♂️